op-scim-bridge/templates/op-scim.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         - name: "OP_SESSION"
           value: "{{ .Values.scimUserHome }}/{{ .Values.scimsessionFile }}"
         - name: "OP_REDIS_URL"
-          value: "redis://{{ .Values.applicationName }}-redis-svc:6379"
+          value: "redis://{{ .Values.applicationName }}-redis-master:6379"
```

Comment:
I'm assuming this new name is defined somewhere in the dependency image, correct?

Reply:
Yes. That is what I gathered too. As a test I deployed it and saw what the name was of the redis master.

I'll be including additional config options and reasonable defaults when I work on #21 next and this will likely also have some values for redis.